kml/serdes_gpx.cpp

```diff
@@ -334,6 +340,24 @@ void GpxParser::ParseAltitude(std::string const & value)
     m_altitude = geometry::kInvalidAltitude;
 }
 
+LocalizableString GpxParser::BuildDescription(LocalizableString description, LocalizableString comment)
+{
+  if (description.empty())
+  {
+    return comment;
+  }
+  else if (comment.empty())
+  {
+    return description;
+  }
+  else
+  {
+    LocalizableString result = LocalizableString();
+    result[kDefaultLang] = description[kDefaultLang] + "\n\n" + comment[kDefaultLang];
+    return result;
+  }
+}
```

Comment:
Do we really need to use Localizable strings in gpx parsing? They were used for multilang kml serialization.

Reply:
Replaced them with strings